src/applications/hca/tests/00-all-fields.e2e.spec.js

```diff
@@ -223,9 +223,6 @@ module.exports = E2eHelpers.createE2eTest(client => {
     .to.not.contain('/review-and-submit')
     .before(Timeouts.slow);
 
-  // Submit message
```

Comment:
this rule can be put back when the original ConfirmationPage component is restored.

Reply:
e2e test restored for original page; commit pushed up.

Now e2e test file no longer appears under Files-changed.